### ISSUE TITLE
Feature addition: hot dog colour scheme

### ISSUE THREAD

**Issue by davetyson** [davetyson/hotDogBot2#4]
I thought it would be fun to add a new colour scheme that makes the page look like a hot dog:

* background bun coloured (light enough that black text still gives accessible contrast ratio over-top
* chat window pink like a hot dog
* message bubbles ketchup and mustard coloured
* buttons and extra touches green like relish, or use one of the other colours, I can play around with it
